setup/install/halyard.md

```diff
@@ -26,6 +26,9 @@ There are a few different ways to install Halyard (with more on the way):
 
 ### Ubuntu 14.04/16.04
 
+Note: While Halyard is supported on Ubuntu 16.04 the current release of Spinnaker
+is not.  It's possible to run Spinnaker's nightly builds but that path is not validated.
```

Comment:
Not sure what this 2nd sentence means. Is it necessary, or should we just say "not supported" for now?

Reply:
Yup, agreed.